content/help/ops/security/authorization-permissive/index.md

```diff
@@ -4,14 +4,17 @@ description: Authorization is enabled, but requests make it through anyway.
 weight: 50
 ---
 If authorization checks are enabled for a service and yet requests to the service aren't being blocked, then
-its likely that authorization was not actually enabled successfully. You can do the following to verify:
+it's likely that authorization was not actually enabled successfully. You can do the following to verify:
```

Comment:
Great catch but I would write: it is likely ...
This contraction is ambiguous otherwise, since it could be read as both "is" or "was".
I would write:
authorization was likely not enabled successfully. To verify, follow these steps: